docs/reference/compatibility.rst

```diff
@@ -1,20 +1,48 @@
 Compatibility
 *************
 
+
+
+System requirements
+~~~~~~~~~~~~~~~~~~~
+
+
 .. only:: iso
 
     The table below shows the system requirements of version |version_major| of the Inmanta Service Orchestrator.
 
+
 .. only:: oss
 
     The table below shows the system requirements of Inmanta OSS version |release|.
 
-
 .. datatemplate:json:: /reference/compatibility.json
    :template: system_requirements.tmpl
 
+
+
 .. only:: iso
 
+    Inmanta core and extensions versions
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
     .. datatemplate:json:: /reference/compatibility.json
        :template: components_requirements.tmpl
 
+
+Module set
+~~~~~~~~~~
+
+
+.. only:: iso
+
+    The table below shows the set of Inmanta modules that come along with version |version_major| of the
+    Inmanta Service Orchestrator and constraints on their respective version (if any).
```

Comment:
I'm not 100% sure about their meaning and the information we want to convey here, what about:
"These constraints ensure compatibility among the modules within this version of the Inmanta Service Orchestrator." ?

Reply:
I am not sure about this myself. Internally we use these constraints to make sure we don't release breaking changes within a major release. But I am not sure what information this represent for our customers. @bartv Do you know?

Reply:
I have no idea either why this is required. Same for all of the extensions but in core it seems to be something really important to add this to the docs. I can accept that but then it also needs to explanation so that the information has meaning or even worse that it does not cause any confusion.

Reply:
I agree. But nobody in the team seems to know why we want this in the docs. @wouterdb Do you?

Reply:
Two reasons why I think we need it for the modules:
1. if a module is developed for a specific ISO version (that is not the latest), these constraints allow you to keep your modules in-range even when only using a single distribution channel (which is the case for us internally). We use this a lot internally, but via the irt repo.  Being able to add these constraints also makes it less dangerous when the wrong distribution channel is configured by accident. 
2. if we get a support request, it allows us to verify that the modules involved are in-range for the orchestrator used. 

For the extensions: less common, we may choose to drop it. It is mostly important when using pytest-inmanta and lsm. Then you need an in-range version of inmanta-lsm but that can now be solved elegantly by installing inmanta-service-orchestrator~=X.0 .